### PR TITLE
Adding a section for support architectures for self-hosted runners

### PR DESCRIPTION
**Pull request by thejoebourneidentity**
### Why:
Unclear to customers what architectures are supported by premium runners

### What's being changed:
The 'about self-hosted runners' page to include a section for support architectures.

### Check off the following:
- [ ] I have reviewed my changes in staging. (look for the **deploy-to-heroku** link in your pull request, then click **View deployment**)
- [ ] For content changes, I have reviewed the [localization checklist](https://github.com/github/docs/blob/main/contributing/localization-checklist.md)
- [X] For content changes, I have reviewed the [Content style guide for GitHub Docs](https://github.com/github/docs/blob/main/contributing/content-style-guide.md).

### CODE REVIEW

File: content/actions/hosting-your-own-runners/about-self-hosted-runners.md [hunk 1]
```diff
@@ -94,6 +94,13 @@ The following operating systems are supported for the self-hosted runner applica
 
 {% if enterpriseServerVersions contains currentVersion %}
 
+#### Architectures
```
Comment (TingluoHuang) [inline content/actions/hosting-your-own-runners/about-self-hosted-runners.md:97]: we don't support x86

arm32 and arm64 are linux only.

we don't support any window-arm

Comment (thejoebourneidentity) [reply content/actions/hosting-your-own-runners/about-self-hosted-runners.md:97]: Fixed

**Review by TingluoHuang — APPROVED** [review]

**Comment by iskandar80** [comment]
Selesaikan dulu dokumen yang mahu diserahkan kepadA saya . Jangan lari dulu ,biar kita mengikut undang undang.

**Comment by martin389** [comment]
A preview is available in staging: https://docs-3270--thejoebourneidentit.herokuapp.com/en/actions/hosting-your-own-runners/about-self-hosted-runners#supported-architectures-and-operating-systems-for-self-hosted-runners ⚡

**Review by martin389 — APPROVED** [review]

**Comment by martin389** [comment]
Thanks for this PR! 🎊 The changes will be visible here once the background publishing steps have completed: https://docs.github.com/en/actions/hosting-your-own-runners/about-self-hosted-runners#architectures

**Pull request merged**